pyqtgraph/graphicsItems/InfiniteLine.py

```diff
@@ -334,18 +327,7 @@ def vsValue(self):
             return self.vs_p[0]
         else:
             return self.vs_p
-        
 
-    ## broken in 4.7
```

Comment:
I actually think we should not only leave this code in, but we should un-comment it (and make it python3 compatible).  It was commented out to deal w/ a bug that I'm fairly sure no longer exists.  But we probably should investigate closer.